src/plugins/condition/components/inspector/StylesView.vue

```diff
@@ -730,10 +737,21 @@ export default {
                     }
                 });
             } else {
-                domainObjectStyles = {
-                    ...domainObjectStyles,
-                    ...objectStyle
-                };
+                if (this.useConditionSetOutputAsLabel === false) {
```

Comment:
The issue @charlesh88 mentioned with widget label on load is I think because this could be `undefined` if `domainObject.configuration.useConditionSetOutputAsLabel` is `undefined`. I also noticed the widget displaying `__no_value` if I exited edit mode without saving.